src/main/resources/META-INF/maven/site.vm

```diff
@@ -52,6 +52,11 @@
 #**##end
 #**##if( $headContent )$headContent#end
 #**##googleAnalytics( $decoration.googleAnalyticsAccountId )
+#**##if (  $decoration.getCustomValue( 'matomo.url' ) && $decoration.getCustomValue( 'matomo.url' ) != ''
+            && $decoration.getCustomValue( 'matomo.siteId' ) && $decoration.getCustomValue( 'matomo.siteId' ) != ''
+  )
+#*  *##matomo( $decoration.custom.getChild('matomo') )
```

Comment:
This test should be pushed into the macro, just like here:
https://github.com/apache/maven-fluido-skin/blob/0cef65c2abd91af3297b2e7ec95790e7b4bfabb0/src/main/resources/META-INF/maven/site-macros.vm#L548-L570